go.mod

```diff
@@ -2,11 +2,19 @@ module github.com/pingcap/failpoint
 
 require (
 	github.com/pingcap/errors v0.11.4
-	github.com/pkg/errors v0.9.1 // indirect
 	github.com/sergi/go-diff v1.1.0
 	github.com/stretchr/testify v1.7.0
 	go.uber.org/goleak v1.1.10
 	golang.org/x/mod v0.17.0
 )
 
-go 1.13
+require (
+	github.com/davecgh/go-spew v1.1.1 // indirect
+	github.com/pkg/errors v0.9.1 // indirect
+	github.com/pmezard/go-difflib v1.0.0 // indirect
+	golang.org/x/lint v0.0.0-20190930215403-16217165b5de // indirect
+	golang.org/x/tools v0.13.0 // indirect
+	gopkg.in/yaml.v3 v3.0.0-20200313102051-9f266ea9e77c // indirect
+)
+
+go 1.18
```

Comment:
```parser``` has used ```go1.19```.

Reply:
i just upgrade it to the minimal version that compiler complains